cmd/blueprints.go

```diff
@@ -825,7 +904,7 @@ func bpCreate(rn string) string {
 			}
 			e := skipExistingHookFile(fileWithPath)
 			if e {
-				fmt.Printf("Skipping processing of hook file %v\n", fileWithPath)
+				//fmt.Printf("Skipping processing of hook file %v\n", fileWithPath)
```

Comment:
Rather than not printing this level of information, create a log that can save it